### ISSUE TITLE
Problem with tensorflow_infer.py

### ISSUE THREAD

**Issue by grdsvt** [PureHing/face-mask-detection-tf2#2]
Hi, I tried to run  tensorflow_infer.py downloaded from AIZOOtech but I dad problems with tensorflow_loader.py.
Did you change something in code?
I have tensorflow 2.1 version with Python 3.7.7.
I have not GPU
Can you help me?
Thanks

**Comment by PureHing** [comment]
tf2.x has no Session, You can use keras_infer.py

**Comment by grdsvt** [comment]
Hi, 
thanks for your answer.
